From dfc5cf657d5a0d5ab659c3f4d5e9b4530850642e Mon Sep 17 00:00:00 2001
From: Vassil Vassilev <v.g.vassilev@gmail.com>
Date: Wed, 18 Oct 2017 17:26:01 +0200
Subject: [PATCH 09/27] The annotation token became a special one.

We can only consume it with calling a special interface or calling ConsumeAnyToken.
---
 lib/Interpreter/LookupHelper.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/Interpreter/LookupHelper.cpp b/lib/Interpreter/LookupHelper.cpp
index b28370cf..4df32ba5 100644
--- a/lib/Interpreter/LookupHelper.cpp
+++ b/lib/Interpreter/LookupHelper.cpp
@@ -753,7 +753,7 @@ namespace cling {
                                              P.getCurToken().getAnnotationRange(),
                                              SS);
       if (SS.isValid()) {
-        P.ConsumeToken();
+        P.ConsumeAnyToken();
         if (!P.getCurToken().is(clang::tok::identifier)) {
           return 0;
         }
-- 
2.14.1

